src/components/Profile/GrantBeneficiaryBox.tsx

```diff
@@ -1,8 +1,8 @@
 import useFormatMessage from '../../hooks/useFormatMessage'
 import useGrantsByUser from '../../hooks/useGrantsByUser'
 
+import { ActionBox } from './ActionBox'
```

Comment:
We should move ActionBox to a common folder now as we use it in both home and profile pages.